.gitignore

```diff
@@ -33,3 +33,6 @@
 
 # Compiled decode utility
 decode_binary
+
+# Development files
```

Comment:
You can ignore things like this in `.git/info/exclude` instead, in case others wont have the same issue.  Not a problem to add it here, of course.